src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -1092,6 +1093,25 @@ describe("hubConnection", () => {
         }
     });
 
+    it("sets the user agent header", async (done) => {
+        const hubConnection = getConnectionBuilder(HttpTransportType.LongPolling, TESTHUB_NOWEBSOCKETS_ENDPOINT_URL)
```

Comment:
Why use the `TESTHUB_NOWEBSOCKETS_ENDPOINT_URL` if WebSocketTransport.ts was changed to send the header?

Reply:
`<grumble>` because it requires some browser checks in the test `</grumble>`

I'll update it.